docs/public/assets/bundle.js

```diff
@@ -27765,7 +27765,7 @@ module.exports = require('./lib/React');
 },{"./lib/React":146}],279:[function(require,module,exports){
 module.exports={
   "name": "ui-toolkit",
-  "version": "0.25.0",
```

Comment:
I'm not sure why this has updated, this should stay at `0.25.0` unless someone hasn;t re-built on master properly